cumulus/test/service/src/cli.rs

```diff
@@ -51,11 +54,36 @@ pub struct TestCollatorCli {
 	#[arg(long)]
 	pub fail_pov_recovery: bool,
 
-	/// EXPERIMENTAL: Use slot-based collator which can handle elastic scaling.
+	/// DEPRECATED: This feature has been stabilized, pLease use `--authoring slot-based` instead.
 	///
+	/// Use slot-based collator which can handle elastic scaling.
 	/// Use with care, this flag is unstable and subject to change.
 	#[arg(long)]
 	pub experimental_use_slot_based: bool,
+
+	/// Authoring style to use.
+	#[arg(long, default_value_t = AuthoringStyle::Lookahead)]
+	pub authoring: AuthoringStyle,
+}
+
+/// Collator implementation to use.
+#[derive(PartialEq, Debug, ValueEnum, Clone, Copy)]
+pub enum AuthoringStyle {
```

Comment:
nit: Would it be worth to avoid duplication of this enum? super-nit: And maybe policy would be better phrasing?